command.c

```diff
@@ -1266,11 +382,19 @@ void redis_parse_cmd(struct cmd *r) {
         }
     }
 
-    len = _scnprintf(
-        r->errstr, 100,
-        "Parse command error. Cmd type: %d, state: %d, break position: %d.",
-        r->type, state, (int)(p - r->cmd));
-    r->errstr[len] = '\0';
+    if (info != NULL && info->subname != NULL)
+        snprintf(r->errstr, 100, "Failed to find keys of command %s %s",
```

Comment:
Nice with useful and detailed error info!

Maybe we should have a define for the size 100 used here and in `hi_malloc` above

Reply:
In that case, we should also move these strings to defines, since the number depends on the length of the messages.

I think we can keep it here. (The reader doesn't need to jump to a different place to see what the code does.)

Reply:
I can put 100 in a local variable at least. That makes it one place to change.